### ISSUE TITLE
Sparse convolution seems to output something different than standard convolution

### ISSUE THREAD

**Issue by zhouxingyu13** [csmliu/AdaDSR#10]
Thank you for your work, I am trying to run your code. In my testing, I found that the performance of sparse convolution seemed to be much worse than that of standard convolution, and the output of single-layer neural network seemed to be always deviated from the correct value of the output of standard convolution. I suspected that this was the reason why the performance of sparse convolution was inferior to that of standard convolution. I don't know if this is a problem with my own compiled code, or if you have produced such problems in your experiments.

**Comment by csmliu** [comment]
In my experiments, I found that the sparse convolution yields the same result as the dense convolution. But you need to set the mask as all 1. The main reason for worse results is the training of RCAN (and AdaRCAN), I can't get the same results as officially released models, even with the official training code.
